frontend/src/shared/components/SearchInput.tsx

```
@@ -1,26 +1,27 @@
 
 // 나중에 로직 안 붙였어요.
 
-const SearchInput = ({ onSearch }: SearchBarProps) => {
+const SearchInput = ({ onSearch, placeholder }: SearchBarProps) => {
   const { keyword, handleChange } = useSearch({ onSearch });
 
   return (
     <form
       role='search'
-      className='mb-6 inline-flex h-12 w-full items-center justify-between rounded-lg bg-gray-50 px-5 py-3.5'
+      className='mb-6 inline-flex h-12 w-100 items-center justify-between rounded-lg bg-gray-50 px-5 py-3.5'
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<search>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)